beps/bep_0003.rst

```diff
@@ -87,70 +92,115 @@ announce
 
 info
   This maps to a dictionary, with keys described below.
+  
+``piece layer``
```

Comment:
Yes, each list entry represents one file.

No, it does not represent full trees, that part. It is meant to be only one level of the tree where the data size covered by the hash is equal to the piece length. so if piece length = 16KiB you get leaf hashes, if it's 32KiB then you get 1 level up, if it's 64KiB you get two levels up etc.

I would prefer always having the 16KiB leaves in there but some users *today* scale their piece length to keep the .torrent file very small, so I assume they would dislike the potentially massive increase in torrent sizes.